### ISSUE TITLE
Return a result even when MetricMDS does not converge

### ISSUE THREAD

**Issue by joeldierkes** [JuliaStats/MultivariateStats.jl#233]
Three algorithms can throw a `ConvergenceException`: `MetricMDS`, `ICA` and `PPCA`. The exception contains information about the convergence criteria and how they are unrealized, without returning an actual model.

While this is the desired default behavior, it would be nice to have the option to opt out of the exception and return a model even if the convergence criteria are not met.

An application would be a repeated fit of such a model on different datasets, where the actual convergence is not as important for some individual datasets as the successful execution of the fit. The only options right now are to either exclude such a dataset or set individual convergence criteria, which is quite cumbersome.

**Comment by l8l** [comment]
I agree that this would be nice to have. Instead of an error, the user could be shown a warning.
